### PR TITLE
Update the missing repositories for repoman

### PR DESCRIPTION
**Pull request by rgolangh**
Add the repositories needed for repoman and its dependencies.

### CODE REVIEW

**Comment by yanirq** [comment]
+1 Verified (Works for me on EL7)

**Comment by rgolangh** [comment]
Verified: works for me on bootstrapped el7

**Comment by eedri** [comment]
ci test please

**Comment by gbenhaim** [comment]
ci test please

**Comment by rgolangh** [comment]
Refined the README to include the repo configuration.

File: README.rst [hunk 1]
```diff
@@ -37,14 +37,19 @@ For EL distros (such as CentOS, RHEL, etc.)::
     enabled=1
     gpgcheck=0
     
-For EL7 distors include those repositories for dependency::
+For EL7 distors include those repositories for dependency:
```
Comment (mykaul) [inline README.rst]: distors -> distros

File: README.rst [hunk 2]
```diff
@@ -37,14 +37,19 @@ For EL distros (such as CentOS, RHEL, etc.)::
...
 
     yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 
-And the ci-tools::
-    yum-config-manager --add-repo http://resources.ovirt.org/repos/ci-tools/$release
-    
-note: tweak the gpgcheck for the ci-tools repo if needed.
+ci-tools::
+
+    [ci-tools]
```
Comment (mykaul) [inline README.rst]: perhaps instead of 'ci-tools' it should be 'ovirt-ci-tools' ?

File: README.rst (hunk 1 above)
Comment (rgolangh) [inline README.rst]: done

File: README.rst (hunk 2 above)
Comment (rgolangh) [inline README.rst]: done

**Comment by pcbailey** [comment]
+1 Works for me on EL7.

**Comment by gbenhaim** [comment]
Thanks @rgolangh 
Can you squash the commits ?

**Comment by rgolangh** [comment]
@gbenhaim sure. note to self - try to avoid changing stuff online even though it is tempting :)

**Comment by nvgoldin** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by rgolangh** [comment]
ci test please

**Comment by nvgoldin** [comment]
ci test please

**Comment by nvgoldin** [comment]
closing as this is already addressed in: http://ovirt-system-tests.readthedocs.io/en/latest/docs/general/installation.html

**Pull request closed without merging**